src/main/java/seedu/application/logic/commands/SortByCompanyCommand.java

```diff
@@ -0,0 +1,30 @@
+package seedu.application.logic.commands;
+
+import seedu.application.logic.commands.exceptions.CommandException;
+import seedu.application.model.Model;
+
+/**
+ * Sorts the applications list by company
+ */
+public class SortByCompanyCommand extends SortCommand {
+
```

Comment:
Just wondering, is there a reason to split SortCommand to three section? I was thinking you could also make one SortCommand class and passing the `orderKeyword` till `ModelManager` and do the sorting logic there.

But Perhaps this implementation has better abstraction 🥲